src/main/java/TaskList.java

```diff
@@ -0,0 +1,56 @@
+import java.util.Scanner;
+
+public class TaskList {
+    public static void printList(Task[] taskList, int size) {
+        for (int i = 0; i < size; i += 1) {
+            System.out.print("      ");
+            Task task = taskList[i];
+            System.out.print(i + 1 + ".");
+            task.printTask();
+        }
+    }
+    public static void main(String[] args) {
+        System.out.println("MOBY: Hello! I'm Moby.");
+        System.out.println("MOBY: What can I do for you?");
+        String line;
+        Scanner in = new Scanner(System.in);
+        Task[] list = new Task[100];
+        int currSize = 0;
+        while (true) {
+            System.out.print("YOU: ");
+            line = in.nextLine();
+            if (line.equals("bye")) {
+                break;
+            }
+            if (line.equals("list")) {
```

Comment:
The code in the if-else if-else blocks seem quite lengthy; consider refactoring them into smaller methods